woocommerce/qit-cli
# Fix issue with "--plugins=woocommerce" and "--woo" parameter

As per Slack https://a8c.slack.com/archives/C03R7A7D1DW/p1718530807725489

Fixes an issue on running Custom E2E tests when you used both the `--woo` parameter and `--plugin=woocommerce`. If you used `--plugin woocommerce` it would work.

This PR fixes this behavior, and also prevents the user from using both `--woo` and `--plugin=woocommerce`, and asks to pick one or the other.

### Testing Instructions
- Check out this branch
- Try to run a test with both `--woo <version>` and `--plugin woocommerce` and see that it prevents you from doing that
- Check that the self-tests for the custom tests are passing:
  - _PS: You might need to spin up your Manager or deploy trunk to Staging depending on what environment you have in your `.env` file._
  - `cd _tests/custom-tests`
  - `./vendor/bin/paratest`
